usaspending_api/awards/migrations/0096_removing_subaward_models.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.13 on 2022-08-26 20:10
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('awards', '0095_auto_20220617_1620'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='BrokerSubaward',
+        ),
+        migrations.DeleteModel(
+            name='Subaward',
+        ),
```

Comment:
Just to confirm: once this PR is deployed, there will be no more "ETL'ing" of _any_ subaward data through the usaspending postgres DB ... _except_ for **landing** the final-curated-product of subaward data (`rpt.subaward_search`) from delta; yes?

Reply:
✅